Formula/source-highlight.rb

```diff
@@ -21,7 +21,7 @@ class SourceHighlight < Formula
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--with-boost=#{HOMEBREW_PREFIX}"
+                          "--with-boost=#{Formula["boost"].prefix}"
```

Comment:
Shouldn't we use `Formula["boost"].opt_prefix` here?

Reply:
`Formula["boost"].prefix` gets expanded to `Formula["boost"].opt_prefix` by magic when referring to a formula that's not the current formula.
https://github.com/Homebrew/brew/blob/d5444b62562949edd3b4494341333e1897078c9b/Library/Homebrew/formula.rb#L539-L543

Reply:
`opt_prefix` is standard.

Reply:
Fixed.